stl/src/filesystem.cpp

```diff
@@ -224,13 +224,11 @@ namespace {
     }
 
     [[nodiscard]] __std_win_error __stdcall _Translate_not_found_to_success(const __std_win_error _Err) noexcept {
-        switch (_Err) {
-        case __std_win_error::_File_not_found:
-        case __std_win_error::_Path_not_found:
+        if (__std_is_file_not_found(_Err)) {

```

Comment:
Now this is additionally translating `__std_win_error::_Invalid_name`. That seems desirable, but is it possible to construct a test case for this?

Reply:
I don't know of one. Every bogus path I was able to form is coming back with the netpath one :(. I observe the _Invalid_name one was added in https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/msvc/commit/4110cc61e2e22dc826c2ba517a237dc76134ab63 -- it might have just been emailed to me or something that someone saw it somewhere.

Even without test cases I don't think there's good reason for variation on what we consider not found.